.gitignore

```diff
@@ -0,0 +1,2 @@
+#dependencies
```

Comment:
- [ ] Let's remove node_modules in your repo

- Just go ahead and delete that folder on GitHub first
- Delete the same directory in your local project
- clean your cache by: `npm cache clean --force`
- Add and commit your changes.
- Push again
- Your problem may be resolved.

Reply:
When I run the command it says `npm WARN using --force Recommended protections disabled.`